app/src/gql/apollo.js

```diff
@@ -15,13 +15,34 @@ import { split } from "apollo-link"
 import { getMainDefinition } from "apollo-utilities"
 import VueApollo from "vue-apollo"
 import { getGraphqlHost } from "scripts/url"
-import * as Sentry from "@sentry/browser"
 import config from "src/../config"
 import introspectionQueryResultData from "src/../fragmentTypes.json"
 import { getFingerprint } from "scripts/fingerprint"
 
 Vue.use(VueApollo)
 
+// to avoid circular dependencies we use this error handler, for errors happening in the router
+// those errors should trigger things in the store (that needs the router)
+class DeferredErrorHandler {
+  constructor() {
+    this.errors = []
+  }
+  error(error) {
+    if (this.errorCallback) {
```

Comment:
this will be true pretty much all the time after the first error is triggered, won't it?

Reply:
ah ok, actually looks good yes. Because this is the callback you are passing in initializeApp.js. Slick

```
  routerErrorHandler.onError((error) => {
    if (error.extensions && error.extensions.code === "UNAUTHENTICATED") {
      store.dispatch("signOutUser")
      return
    }
    // if sentry is enabled pass all error directly to sentry
    if (config.sentryDSN) {
      // pass errors to sentry
      Sentry.captureException(error)
    } else {
      console.error(error)
    }
  })
```